frontend/packages/dev-console/src/components/__tests__/ProjectsExistWrapper.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import ProjectsExistWrapper from '../ProjectsExistWrapper';
+
+const projects: any = {
+  data: [],
+  loaded: true,
+};
+
+describe('ProjectsExistWrapper', () => {
+  it('should render projects exist wrapper component', () => {
+    const wrapper = shallow(
+      <ProjectsExistWrapper title="Topology" projects={projects}>
+        {}
+      </ProjectsExistWrapper>,
+    );
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should show loading box', () => {
+    const data = _.cloneDeep(projects);
```

Comment:
It might be better to define default project props in beforeEach and have necessary overrides in each test case.